__tests__/create-object/index.test.js

```diff
@@ -0,0 +1,14 @@
+import createObject from '../../functions/create-object/1.0';
+
+describe('Create object', () => {
+  test('It creates an object with the given properties', async () => {
```

Comment:
test also the usecase what will happen if the key or value is empty so the user keeps them empty. Try to catch this behaviour in your code